### ISSUE TITLE
Slider is not focusable when navigating with a keyboard

### ISSUE THREAD

**Issue by gmoraja** [flutter/flutter#48905]
@goderbauer
@gspencergoog
@lorenzOliveto

Steps to reproduce:
- Create a simple app with a Slider in the center
- Try to navigate the app with a physical keyboard

You can find an example in this repository, https://github.com/lorenzOliveto/flutter_keyboard_navigation_issues (slider_example.dart)

**Comment by gspencergoog** [comment]
Correct, sliders do not yet have focus support.  This is just an overlooked feature that needs to be implemented still.

**Comment by clocksmith** [comment]
In addition to making the slider focusable, it should also have the appropriate keyboard actions

**Comment by andraskende** [comment]
+1
Please also add android TV D-pad support

**Comment by gmoraja** [comment]
@gspencergoog on a mobile device, the slider can be browsed with an external keyboard now, but it doesn't receive focus with TalkBack yet.  Should I open a separate issue?

**Comment by gspencergoog** [comment]
@clocksmith are you also adding Semantics focus/increment/decrement information to the sliders?

**Comment by clocksmith** [comment]
@gspencergoog the slider does work with TalkBack already. It reports its current value, and it can be modified with the volume keys.

@gmoraja If you have a new issue with TalkBack, yes, please open a separate issue because this one was for navigating with keyboard.

**Comment by lorenzOliveto** [comment]
@gspencergoog @clocksmith @gmoraja 
Yes, the slider has the `semanticFormatterCallback` property set and it works correctly if the app is navigated with TalkBack (swipe left and swipe right).
The problem is that when you have Talkback on and you're navigating the app with a keyboard (with the tab key) the accessibility focus (the green rectangle) doesn't move when the Slider receives the keyboard focus and remains in the previous element.
Should we open a separate issue for this even if it's connected with this?
Here's a video and Flutter Doctor output:
[m-pro-keyboard.mp4.zip](https://github.com/flutter/flutter/files/4556670/m-pro-keyboard.mp4.zip)

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel master, 1.18.0-9.0.pre.73, on Mac OS X 10.15.4 19E287, locale it-IT)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[✓] Xcode - develop for iOS and macOS (Xcode 11.4)
[✓] Android Studio (version 3.6)
[✓] Connected device (1 available)

• No issues found!
```
